test/tree_stand/node_test.rb

```diff
@@ -51,6 +51,22 @@ def test_can_navigate_to_the_parent_nodes
     assert_equal(@tree.root_node, node.parent.parent)
   end
 
+  def test_can_enumrate_named_children
```

Comment:
test_can_enumerate_named_children

(I'm not provided with means to suggest edits.)